scripts/circle-ci-tests.sh

```diff
@@ -77,9 +76,9 @@ function get_children() {
             lms.djangoapps.branding) ;&
             lms.djangoapps.ccx) ;&
             openedx.core.djangoapps.cors_csrf) ;&
-            openedx.features.course_experience) ;&
+            openedx.features.course_experience)
             # SEG_FAULT
-            lms.djangoapps.mobile_api)
+            # lms.djangoapps.mobile_api)
```

Comment:
This passes now.

Reply:
why comment this out rather than just delete the line?

Reply:
Because I wasn't sure if it would break on Circle or not..

Reply:
gotcha